transformers/modeling_bert.py

```diff
@@ -601,7 +653,7 @@ def forward(self, input_ids, attention_mask=None, token_type_ids=None, position_
         # positions we want to attend and -10000.0 for masked positions.
         # Since we are adding it to the raw scores before the softmax, this is
         # effectively the same as removing these entirely.
-        extended_attention_mask = extended_attention_mask.to(dtype=next(self.parameters()).dtype) # fp16 compatibility
+        extended_attention_mask = extended_attention_mask.to(dtype=next(self.parameters()).dtype)  # fp16 compatibility
```

Comment:
Same here, try to avoid these, harder to pin-point real changes when reviewing PRs.

Reply:
Ok, I'll revert all the whitespace / linebreak changes.